sp800-63-3/sec5_xAL.md

```diff
@@ -31,14 +33,14 @@ A summary of each of the identity, authenticator, and federation assurance level
 
 | Identity Assurance Level |
 |:----------------------|
-| **IAL1** - At this level, attributes provided in conjunction with the authentication process, if any, are self-asserted.|
+| **IAL1** - At this level, attributes, if any, are self-asserted or should be treated as self-asserted.|
```

Comment:
I meant that the other occurrences also need to change (IAL2 to IAL 2 and IAL3 to IAL 3)